portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase

```diff
@@ -1058,9 +1060,13 @@ definition {
 	@description = "Documents visitor behavior card shows expected amount of views"
 	@priority = "5"
 	test VisitorBehaviorCardShowsExpectedAmountOfPreviewsInDM {
-		property portal.acceptance = "true";
+
+		//property portal.acceptance = "true";
+
 		property portal.release = "true";
```

Comment:
Thanks for finding this! Do you think we should also comment out the portal.release run as well?

Reply:
Hey Josh, I've checked the last 4 runs of tests that we have from the portal.release of the new DXP U release and those tests are constantly passing there. I believe we can leave this property, but I'm wondering why the problem doesn't happen there 🤔 😆